### ISSUE TITLE
Remove code out of p5 editor and into the library

### ISSUE THREAD

**Issue by MathuraMG** [MathuraMG/p5-accessibility#12]
css and structure of html with accessibility could/should be done in the library and not in [PreviewFrame.jsx](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/IDE/components/PreviewFrame.jsx)

**Comment by lm-n** [comment]
#14 adds CSS to library. HTML structure was not added upon discussion with regards to the flexibility of the library. We should talk more about this @MathuraMG

**Comment by lm-n** [comment]
https://github.com/lm-n/p5.js-web-editor/tree/accessibility has been updated: removed CSS, added aria-label, include preliminary CDN
